### PR TITLE
Add readme top with gifs

### PR DESCRIPTION
**Pull request by chrisplatorres**
This PR proposes adding the top part of the README with related exokit-web gifs.

Adding the below gifs:

## First half 

webgiff
worldmeshgif
MMMm6-min

-----------------------------------------------
## Second half 

hubscvgif5
hubscvgif8
MMMm8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <a href="https://twitter.com/exokitxr">Twitter</a>
+  &mdash;
+  <a href="https://mailchi.mp/ee614096d73a/exokitweb">Email List</a>
+  &mdash;
+  <a href="https://twitter.com/exokitxr"><img src="https://img.shields.io/twitter/follow/exokitxr.svg?style=social"></a>
+</div>
+
+## Examples
+
+<a href="https://web.exokit.org/"><img alt="Hands Reality Tab" target="_blank" src="https://user-images.githubusercontent.com/29695350/64732206-32008580-d4a8-11e9-994c-a0e18c662b72.gif" height="190" width="32%"></a>
```
Comment (chrisplatorres) [inline README.md]: Need to fix img alts

**Comment by modulesio** [comment]
Changed the base to `readme` -- could you resolve?

**Comment by chrisplatorres** [comment]
This is ready to merge into `readme`.

**Comment by chrisplatorres** [comment]
Ready to merge now.

**Pull request merged**